src/layer/utils.ts

```diff
@@ -80,3 +80,9 @@ export function parseSHInstanceId(baseUrl: string): string {
   }
   throw new Error(`Could not parse instanceId from URL: ${baseUrl}`);
 }
+
+export let debugEnabled = false;
```

Comment:
This could probably be moved to some better location as this debug switch is not limited to layers only, but it can be used for all axios request. fftr

Reply:
Good point! I have moved it now to `utils/debug.ts`. 

While at it, I have also removed access to `debugEnabled` variable and implemented a (trivial) getter.